dark/index.html

```diff
@@ -224,13 +224,17 @@ <h3>ABOUT <span class="text-info">ME</span></h3>
           subject is Information and Communication Technology. My mother's name is Sudeshika Manthilake and my father's
           name is Athula Rathnayake. I hope to be a Computer Engineer in the future.
         </p>
+        
       </section>
 
       <!-- Contact Section -->
 
       <section id="contact-section">
             <h1>CONTACT ME</h1>
             <h3>CONTACT <span class="text-info">ME</span></h3>
+            <p class="about-text">Name:  Yoshitha Rathnayake

```

Comment:
Align them correctly because the sentence is in the top of the contact section topic